src/Adaptivity/AdaptedTriangulations.jl

```diff
@@ -0,0 +1,316 @@
+
+"""
+  AdaptedTriangulation
+
+  Triangulation produced from an AdaptedDiscreteModel.
+  
+  Contains: 
+
+  - adapted_model ~> AdaptedDiscreteModel for the triangulation.
+  - trian ~> Triangulation for the background model, i.e get_model(adapted_model).
```

Comment:
Here you actually mean that `trian` is to be extracted out of the background model, correct?

I.e., `trian` is not necessarily `Triangulation(get_model(adapted_model))`

I would rephrase this comment to clarify the point.

Reply:
What I mean is that `trian=Triangulation(get_model(adapted_model),args...;kwargs...)` depending on what type of Triangulation it is.